test_googleMapsApiModule.py

```diff
@@ -2,8 +2,10 @@
 import config
 import pytest
 from googlemaps.exceptions import ApiError
+import unittest 
+
  
-class TestClass:
+class TestClass(unittest.TestCase):
```

Comment:
The current implementation of testing uses `pytest`, please use the same.

Reply:
@vishakha-lall I've changed the code. Please have a look.